src/layout/questionnaire-nav/components/arbo-simplifield.jsx

```diff
@@ -49,23 +49,18 @@ class ArboSimplified extends Component {
             className={isQuestion(components[key]) ? 'questions' : ''}
           >
             {components[key].children && components[key].children.length > 0 && (
-              <a
-                onClick={e => this.handleExpand(e, key)}
-                href="#"
-                aria-label="expand/collapse"
-              >
                 <span
+                  onClick={e => this.handleExpand(e, key)} 
```

Comment:
You can have an onClick on focusable element like link or button. 
You should never have onClick on span .